Terminal.Gui/Drivers/V2/InputProcessor.cs

```diff
@@ -179,6 +180,22 @@ internal bool IsValidInput (Key key, out Key result)
         if (_highSurrogate > 0 && char.IsLowSurrogate ((char)key))
         {
             result = (KeyCode)new Rune (_highSurrogate, (char)key).Value;
+
+            if (key.IsAlt)
+            {
+                result = new (result.KeyCode | KeyCode.AltMask);
```

Comment:
`result = result.WithAlt`

Reply:
Done in [14c68e5](https://github.com/gui-cs/Terminal.Gui/pull/4225/commits/14c68e550f04bce301b1e297d93e65ea1fc76070)